README.md

```diff
@@ -41,24 +41,22 @@ Profiling in a production environment is not recommend.
 
 ### Drupal 8+
 
-The `xhgui/examples` contains files which can be used to quick-start a Drupal installation.
+The `xhgui/examples` contains files to quick-start a Drupal installation configuration.
 
 - Copy the files from `xhgui/examples` to the sites's `web/sites/default` folder.
 
-- Add the following line to `web/sites/default/settings.php` to include the collector.
+- Add the following line to `web/sites/default/settings.local.php` to include the collector.
 
    ```php
    require_once __DIR__ . '/xhgui.collector.php';
    ```
```

Comment:
Instead of requiring this manual configuration, why don't we suggest `require_once /mnt/ddev_config/xhgui/examples/xhgui_collector.php`, then they don't have to find any files or copy them. Beyond that, consider adding the configuration to settings.php the way it's done with ddev/ddev-redis. 

Add this to settings.php:
```php
if (file_exists("/mnt/ddev_config/xhgui/examples/xhgui.collector.php")) {
  require_once "/mnt/ddev_config/xhgui/examples/xhgui.collector.php";
}
```

But if you don't take this path, remember that `settings.local.php` is not enabled in default Drupal, it has to be uncommented in settings.php.

Reply:
The thinking was for the framework to take ownership of the files, therefore copy them.

Going with `require_once "/mnt/ddev_config/xhgui/examples/xhgui.collector.php";` since functions similar to other addons.

All framework guides were updated in abd74e1d7be8eb7275d1a224ee2fa455a9e02609
Return to `settings.php` for Drupal in eeaf13128e7149179b477d1ccfeb48dea18d939a